Formula/s/sprocket.rb

```diff
@@ -0,0 +1,89 @@
+class Sprocket < Formula
+  desc "Bioinformatics workflow engine built on the Workflow Description Language (WDL)"
+  homepage "https://sprocket.bio"
+  url "https://github.com/stjude-rust-labs/sprocket.git",
+    tag:      "v0.13.0",
+    revision: "9ce64989016df93637cb4bbe9a1bddd951795155"
+  license any_of: ["Apache-2.0", "MIT"]
+
+  head "https://github.com/stjude-rust-labs/sprocket.git", branch: "main"
+
+  livecheck do
+    url :stable
+    strategy :github_releases
+  end
+
+  depends_on "rust" => :build
+  depends_on "openssl@3"
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    wdl = <<~WDL
+      version 1.2
+
+      task say_hello {
+        input {
+          String greeting
+          String name
+        }
+
+        command <<<
+          echo "~{greeting}, ~{name}!"
+        >>>
+
+        output {
+          String message = read_string(stdout())
+        }
+
+        runtime {
+          container: "ubuntu:latest"
+        }
+      }
+
+      workflow main {
+        input {
+          String name
+          Boolean is_pirate = false
+        }
+
+        Array[String] greetings = select_all([
+          "Hello",
+          "Hallo",
+          "Hej",
+          (
+            if is_pirate
+            then "Ahoy"
+            else None
+          ),
+        ])
+
+        scatter (greeting in greetings) {
+          call say_hello {
+            input: greeting = greeting, name = name
+          }
+        }
+
+        output {
+          Array[String] messages = say_hello.message
+        }
+      }
+    WDL
+    (testpath/"hello.wdl").write(wdl)
+
+    output = shell_output(
+      "#{bin}/sprocket inputs #{testpath}/hello.wdl"
+    )
+
+    expected = <<~JSON.strip
+      {
+        "main.name": "String",
+        "main.is_pirate": false
+      }
+    JSON
+
+    assert_match expected, output
```

Comment:
this would be simpler as
```suggestion
    expected = <<~JSON.strip
      {
        "main.name": "String",
        "main.is_pirate": false
      }
    JSON

    assert_match expected, shell_output("#{bin}/sprocket inputs #{testpath}/hello.wdl")
```